src/xinterpreter.cpp

```diff
@@ -427,7 +419,7 @@ namespace xcpp
     {
         nl::json kernel_res;
         std::string exp = R"(\w*(?:\:{2}|\<.*\>|\(.*\)|\[.*\])?)";
-        std::regex re(R"((\w*(?:\:{2}|\<.*\>|\(.*\)|\[.*\])?)(\.?)*$)");
+        std::regex re{"(" + exp + R"(\.?)*$)"};
         auto inspect_request = is_inspect_request(code.substr(0, cursor_pos), re);
```

Comment:
In the case of `?std::vector` (as an example), the kernel will have shut down with an error in `process_code` before reaching this line

Reply:
I made this change considering the variable `exp` was defined but not being used throughout the codebase and I think the purpose of defining it was so that it could be used for the regex.
But after going through your comment above, I am not sure what prompt should even trigger this function. 
I thought something like `?std::vector` would end up triggering this rather than entering the execute request code block.